apps/wizarr-frontend/src/modules/admin/components/Forms/InvitationForm.vue

```diff
@@ -184,6 +184,10 @@ export default defineComponent({
                         label: "Access to Live TV",
                         value: "live_tv",
                     },
+                    hide_user: {
+                        label: "Hide User from Dashboard",
```

Comment:
This doesn't make much sense for most users.

I would say it should read "Hide User from the Login Page"

Reply:
Updated to new text